articles/container-apps/tutorial-scaling.md

```diff
@@ -291,6 +295,156 @@ The following screenshot shows a zoomed view of how the requests received by you
 
 :::image type="content" source="media/scale-app/azure-container-apps-scale-replicas-metrics-3.png" alt-text="Screenshot of container app metrics graph, showing requests split by replica, in a zoomed view.":::
 
+## CPU and memory scaling
+
+You should prefer [HTTP scale rules](/azure/container-apps/scale-app#http) to CPU or memory scale rules when possible.
```

Comment:
We should probably remove this unless we want to explain in detail why (can't scale to 0, etc)